### PR TITLE
MP-7786 - Currency Number Db configuration

### PR DESCRIPTION
**Pull request by denisvieira-dev**

### CODE REVIEW

**Comment by gjorgji-grgovski-authoritypartners** [comment]
The title of the PRs should start with "MP-XXXX ........."

**Comment by gjorgji-grgovski-authoritypartners** [comment]
The converter should use the "FromDatabase" method.

File: src/Nox.Types.EntityFramework/Types/CurrencyNumber/CurrencyNumberDatabaseConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState,
+        EntityTypeBuilder builder,
+        NoxSimpleTypeDefinition property,
+        Entity entity,
+        bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(property.IsRequired)
+            .HasConversion<CurrencyNumberConverter>();
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types.EntityFramework/Types/CurrencyNumber/CurrencyNumberDatabaseConfiguration.cs:22]: Converters should use "FromDatabase"

**Review by gjorgji-grgovski-authoritypartners — CHANGES_REQUESTED** [review]

File: src/Nox.Types.EntityFramework/Types/CurrencyNumber/CurrencyNumberDatabaseConfiguration.cs (hunk 1 above)
Comment (denisvieira-dev) [reply src/Nox.Types.EntityFramework/Types/CurrencyNumber/CurrencyNumberDatabaseConfiguration.cs:22]: Done

**Review by rochar — APPROVED** [review]

**Review by danield-ap — APPROVED** [review]

**Pull request merged**